tests/test_martingale.py

```diff
@@ -0,0 +1,51 @@
+from unittest import TestCase
+
+from unittest.mock import Mock, patch
+
+from roulette import Game, Martingale, Table, Wheel, Bet, BinBuilder
+
+
+class TestMartingale(TestCase):
+    def setUp(self):
+        self.table = Table()
+        self.wheel = Wheel()
+        bin_builder = BinBuilder()
+        bin_builder.buildBins(self.wheel)
+        self.game = Game(self.wheel, self.table)
+        self.martingale = Martingale(self.table)
+
+    def test_placeBets_make_bets_if_enough_stake(self):
```

Comment:
we don't test functions rather behavior. so a better name can be `test_placing_bets_need_minimum_stake`.

Reply:
This test can cover both cases (enough stake/less stake)

Reply:
Done.